bucket/honeyview.json

```diff
@@ -1,17 +1,29 @@
 {
+    "homepage": "https://www.bandisoft.com/honeyview/",
     "version": "5.31",
     "description": "A fast image viewer.",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.bandisoft.com/honeyview/eula/"
     },
-    "url": "https://dl.bandisoft.com/honeyview/HONEYVIEW-PORTABLE.ZIP",
-    "homepage": "https://www.bandisoft.com/honeyview/",
-    "hash": "1c6f6184838bf6f62ebcf994c2ec17dbe9e5f111672eb988f6b48d80dc11e392",
-    "bin": "Honeyview32.exe",
+    "architecture":{
+        "64bit":{
+            "url": "https://dl.bandisoft.com/honeyview/HONEYVIEW-SETUP.EXE#/dl.7z",
```

Comment:
This file contains both architectures.

First one (smaller) is 32 bit and second is 64bit.

![image](https://user-images.githubusercontent.com/13260377/63073506-688ac500-bf29-11e9-9cac-8e9c9362f675.png)

Reply:
Yes, but I don't known how to extract the file with same names separately in powershell.

Reply:
I will provide you fixed manifest later.